### DISCUSSION TITLE
I think it would make sense for `diff` and `deploy`to indicate termination protection changes of stacks

### DISCUSSION THREAD

**Discussion by azatoth** [aws/aws-cdk#31928] [General]
I noticed that when you are changing termination protection of a stack, neither `diff` nor `deploy` will make any indication of this change.
Wouldn't it be reasonable to have `diff` output whether termination protection has been enabled or disables, and have `deploy` output that this action was in fact made.

On a separate note: I would love to have a feature flag to enable/disable termination protection for all stacks. Currently I'm using an aspect that will set termination protection on each stack.

**Comment by ashishdhingra** [comment]
Replied on https://github.com/aws/aws-cdk/issues/32043#issuecomment-2462945934.
___
Setting `terminationProtection` to `true` (default `false`):
```TypeScript
const app = new cdk.App();

new CdktestStack(app, 'CdktestStack', {
  env: { account: process.env.CDK_DEFAULT_ACCOUNT, region: process.env.CDK_DEFAULT_REGION },
  terminationProtection: true
});
```
doesn't report the change when running `cdk diff`:
```
Stack CdktestStack
Parameters
[+] Parameter BootstrapVersion BootstrapVersion: {"Type":"AWS::SSM::Parameter::Value<String>","Default":"/cdk-bootstrap/hnb659fds/version","Description":"Version of the CDK Bootstrap resources in this environment, automatically retrieved from SSM Parameter Store. [cdk:skip]"}


✨  Number of stacks with differences: 1
```

Appears to be useful feature to report change in `terminationProtection` for a stack.
